src/content/docs/en/pages/devtools/cli/azion-cli/commands/purge/purge.mdx

```diff
@@ -0,0 +1,42 @@
+---
+title: Azion CLI purge
+description: Azion CLI purge allows you to instantly delete the cache of your content at Azion..
```

Comment:
description: Azion CLI purge allows you to instantly delete the cache of your content at Azion.

Reply:
```suggestion
description: Azion CLI purge allows you to instantly delete the cache of your content at Azion.
```